src/ElvAccount.js

```diff
@@ -56,17 +58,35 @@ class ElvAccount {
    *
    * @namedParams
    * @param {number} funds - The amount in ETH to fund the new account.
-   * @cauth {string} accountName - The name of the account to set in it's wallet metadata (Optional)
-   * @cauth {string} tenantAdminsGroup - The tenant admins group ID to set for the user's wallet (Optional)
+   * @param {string} accountName - The name of the account to set in it's wallet metadata (Optional)
+   * @param {string} tenantId - The tenant ID (iten) (Optional)
    * @return {Promise<Object>} - An object containing the new account mnemonic, privateKey, address, accountName, balance
    */
-  async Create({ funds = 0.25, accountName, tenantAdminsAddress }) {
+  async Create({ funds = 0.25, accountName, tenantId }) {
     if (!this.client) {
       throw Error("ElvAccount not intialized");
     }
-    if (tenantAdminsAddress) {
-      if (!Utils.ValidAddress(tenantAdminsAddress)) {
-        throw Error(`Invalid tenant admins address: ${tenantAdminsAddress}`);
+
+    // We don't require the key is part of a tenant (for example when creating a tenant root key)
+    let tenantAdminsAddr;
+    if (tenantId != undefined) {
```

Comment:
just **if truthy** 
`   if (tenantId)
`   
 tenantId is string and we could miss null or ""

Reply:
Fixed both - thank you